release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -603,6 +603,21 @@ $ operator-sdk new <operator_name> --type=helm \
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1874754[*BZ#1874754*])
 
+* The bare metal actuator is deleting the machine object when its underlying
+host is deleted. Machine controllers should never delete a machine object;
+instead, if a machine fails, it should be transitioned to a failed state. The
+user should always be responsible for deleting a machine object manually.
++
+This is a known issue that could cause the following complications:
++
+** The Machine Remediation controller attempts to remediate the machine by
+rebooting the cluster.
+** Only the host name assigned to the machine is tracked. Therefore, if the
+host is deleted and recreated, the cluster might attempt to provision it again.
```

Comment:
These were examples of things that complicate the solution, not symptoms of the problem.